clientbuffer.cpp

```diff
@@ -65,8 +70,6 @@ class CClientBufferMod : public CModule
         {
             if (Args[n].Equals("autoadd", CString::CaseInsensitive))
                 m_bAutoAdd = true;
-            else if (Args[n].StartsWith("timelimit=", CString::CaseInsensitive))
-                m_iTimeLimit = Args[n].Token(1, false, "=").ToInt();
```

Comment:
Please do not remove features or break backwards-compatibility.

Reply:
means you want to keep the general timelimit as is. with the possibility to change client specific timelimit?

Reply:
Yes, I think it would be good to leave the old setting as the default if a per-client one is not specified.